### PR TITLE
Implemented assignment 1 features

### PR DESCRIPTION
**Pull request by corey1994**
Feature 1: Recognize tied games
Feature 2: Add scoreboard
Feature 3: Add ability to play multiple games

Added .gitignore
Enhanced pom.xml to improve build process and avoid compilation errors.

### CODE REVIEW

File: src/main/java/cs/ualberta/cmput402/tictactoe/scoreboard/Scoreboard.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    private Hashtable<Player, Integer> statistics;
+
+    public Scoreboard() {
+        statistics = new Hashtable<Player, Integer>();
+        statistics.put(Player.X, 0);
+        statistics.put(Player.O, 0);
+        statistics.put(Player.NONE, 0);
+    }
+
+    public void printScoreboard() {
```
Comment (markosviggiato) [inline src/main/java/cs/ualberta/cmput402/tictactoe/scoreboard/Scoreboard.java:25]: This method should be complemented to be explicitly according to the requirements, i.e., it must display the number of games won, lost and tied for each player.

**Review by markosviggiato — COMMENTED** [review]
Thanks for the PR. Everything seems to be correctly implemented, but I have one minor comment.

Just to be explicitly according to the feature requirements, the method _printScoreboard()_ (_Scoreboard.java_) should display the number of games won, lost and tied for each player. Although it is obvious that #gamesWonByX = #gamesLostByO, please alter this method to print the number of games won, lost and tied for each player as required.

File: src/main/java/cs/ualberta/cmput402/tictactoe/scoreboard/Scoreboard.java (hunk 1 above)
Comment (corey1994) [reply src/main/java/cs/ualberta/cmput402/tictactoe/scoreboard/Scoreboard.java:25]: This change has been implemented.  Thank you for the feedback.

**Comment by markosviggiato** [comment]
Thanks for addressing the problem.

**Pull request merged**